lib/debezium/decimal.go

```diff
@@ -23,11 +24,14 @@ func EncodeDecimal(value string, scale uint16) ([]byte, error) {
 		return nil, fmt.Errorf("unable to use %q as a floating-point number", value)
 	}
 
-	data := bigIntValue.Bytes()
+	data := bigIntValue.Bytes() // [Bytes] returns the absolute value of the number.
 	if bigIntValue.Sign() < 0 {
 		// Convert to two's complement if the number is negative
-		bigIntValue = bigIntValue.Neg(bigIntValue)
```

Comment:
No need to do this as the `Bytes()` call already uses the absolute value of the number.